### PR TITLE
Updated tests for the detect_language function

### LINKED ISSUE #31: Write unit tests for function 1

### PR DESCRIPTION
**Pull request by schepal**

### CODE REVIEW

**Review by singh-karanpal — CHANGES_REQUESTED** [review]
please add test cases to test file

File: tests/test_nlpsummarize.py [hunk 1]
```diff
@@ -1,50 +1,85 @@
 from nlpsummarize.part_of_speech import get_part_of_speech
 from nlpsummarize.polarity import polarity
+from nlpsummarize import detect_language
```
Comment (singh-karanpal) [inline tests/test_nlpsummarize.py]:
```suggestion
from nlpsummarize.detect_language import detect_language
```

File: tests/test_nlpsummarize.py [hunk 2]
```diff
@@ -1,50 +1,85 @@
...
+        assert detect_language(test_df['spanish_text'][0]) == 'Spanish', 'Incorrect Language'
+
+    english_input_1()
+    test_english_input()
+    test_chinese_input()
+    test_mixture_input()
+    test_polarity_input()
+    test_polarity_count()
+    test_language_textual()
+    test_language_works()
```
Comment (singh-karanpal) [inline tests/test_nlpsummarize.py]: remove all of these from test.py

**Review by singh-karanpal — CHANGES_REQUESTED** [review]
Please have a look at these 2 changes

File: tests/test_nlpsummarize.py (hunk 2 above)
Comment (schepal) [reply tests/test_nlpsummarize.py]:
```suggestion
   
```

**Review by singh-karanpal — APPROVED** [review]
looks good

**Pull request merged**